sources/api/apiclient/src/main.rs

```diff
@@ -84,10 +83,15 @@ struct RawArgs {
 #[derive(Debug)]
 struct RebootArgs {}
 
+/// Stores a vector of user-supplied key-value pairs for the 'set' subcommand.
+#[derive(Serialize, Deserialize)]
+pub struct SetKeyPairSettings {
+    data: Vec<String>,
```

Comment:
nit: maybe something less generic? (this admittedly isn't much better)
```suggestion
    request_payload: Vec<String>,
```